kuma/wiki/search.py

```diff
@@ -286,13 +287,8 @@ def reindex_all(cls, chunk_size=500, index=None, percent=100):
                            for chunk in chunked(indexable, chunk_size)]
             chord_flow(pre_task, index_tasks, post_task).apply_async()
 
-        message = _(
-            'Indexing %(total)d documents into %(total_chunks)d chunks of '
-            'size %(size)d into index %(index)s.' % {
-                'total': total,
-                'total_chunks': total_chunks,
-                'size': chunk_size,
-                'index': index.prefixed_name
-            }
+        message = (
+            f'Indexing {total:,} documents into {total_chunks} chunks of '
+            f'size {chunk_size} into index {index.prefixed_name}.'
```

Comment:
This change is unrelated but it hurt my eyes to see that legacy use of string interpolation. Also, I don't think these kinds of management commands would never need to be translated away from en-US.